test/chunking-form/samples/entry-aliases/_config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+	description: 'alias module dependency inlining',
+	options: {
+		input: {
+			'main1-alias.js': 'main1.js',
```

Comment:
Wondering if you have any ideas how we could make the aliases be the later entry points in order, rather than the first entry points in order as works here?

It's not a big thing, but would be nice to just lock this behaviour down either way.

Reply:
I added a small change so that both the order in the bundle object and the order of facade creation respects the order configured by the user. I.e. the first chunk for a given entry becomes the actual chunk and later ones become facades, and the order between those is preserved.

Previously, later entries would "overtake" earlier entries because loading the later ones would just return the module instance of the earlier modules, making them "load faster" and receive their name earlier.